### ISSUE TITLE
creative inventory

### ISSUE THREAD

**Issue by ixLava** [cabaletta/baritone#1675]
If I used #allowInventory in a creative mode, will it take blocks from the creative mode, or just my 36-block inventory

Thank u so much
## Final checklist
- [x] I know how to properly use check boxes 
- [ ] I have not used any OwO's or UwU's in this issue.

**Comment by ixLava** [comment]
also, can I have 2 accounts to build the same schematic??

**Comment by dominikaaaa** [comment]
You can, use `#buildIgnoreExisting`

**Comment by ixLava** [comment]
command not found buildIgnoreExisting

‫في الأحد، 10 مايو 2020 في 1:48 ص تمت كتابة ما يلي بواسطة ‪Dominika‬‏ <‪
notifications@github.com‬‏>:‬

**Comment by dominikaaaa** [comment]
Update baritone then.

**Comment by Wyatt-Stanke** [comment]
You keep referencing this issue but you never told us the answer to the first question.

> If I used #allowInventory in a creative mode, will it take blocks from the creative mode, or just my 36-block inventory
> 
> Thank u so much
> 
> ## Final checklist
> * [x]  I know how to properly use check boxes
> * [ ]  I have not used any OwO's or UwU's in this issue.

So,  if I used #allowInventory in a creative mode, will it take blocks from the creative mode, or just my 36-block inventory?

_If I sound like idiot saying this don't make fun of me._

**Comment by dominikaaaa** [comment]
No, it won't, which is why I changed the label to a suggestion so Baritone can do that.

**Comment by waterdragon78** [comment]
PLZZ I NEED THIS

**Comment by boehs** [comment]
oy @StijnSimons , you mentioned in another issue you might attempt it, although it was 2 years ago haha. hate to drag you in but if you got time would love to see it. (#532 )

**Comment by TheRedBlueCube3** [comment]
Is this gonna be added?

**Comment by bddvlpr** [comment]
maybe
